apps/weshipit/pages/clients.tsx

```diff
@@ -26,9 +33,11 @@ export default function ClientsPage({ clients }: clientsPageProps) {
       withHeader
       withContainer
       withFooter
-      seoTitle="Clients"
-      seoDescription="Our clients are at the heart of everything we do. We are proud to have worked with a diverse range of businesses and organizations, helping them achieve their goals with our high-quality React Native development services. Visit our /clients page to see a selection of our satisfied clients and learn more about the projects we have completed for them."
       ogImageTitle="Clients"
+      seoTitle={'Clients'}
+      seoDescription={
```

Comment:
instead of dupplicating the content you can use `metadata.title` and `metadata.description`

Reply:
Type 'string | TemplateString' is not assignable to type 'string'.
  Type 'DefaultTemplateString' is not assignable to type 'string'.ts(2322)